app/controllers/categories_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /categories/1 or /categories/1.json
   def show
+    @articles = Article.all.joins(:category).where(categories: { id: params[:id] })
```

Comment:
`.all` should be latest method call, e.g.:
`Article.joins(:category).where(categories: { id: params[:id] }).all`